packages/react-charts/src/components/ChartPoint/ChartPoint.test.tsx

```diff
@@ -17,6 +20,7 @@ test('renders component data', () => {
       title="Average number of pets"
       height={50}
       width={200}
+      titleComponent={<ChartLabel id="test2" />}
```

Comment:
Same here. We must clone the `titleComponent`, so we don't override the dev's given properties